test/ui-test/src/screens/SettingsScreen.py

```diff
@@ -300,3 +308,20 @@ def check_backup_seed_phrase_workflow(self):
 
     def verify_seed_phrase_indicator_not_visible(self):
         verify_not_found(WalletSettingsScreen.BACKUP_SEED_PHRASE_BUTTON.value, "Check that backup seed phrase settings button is visible")
+        
+    def change_user_password(self, oldPassword: str, newPassword: str):
+        get_and_click_obj(ProfileSettingsScreen.CHANGE_PASSWORD_BUTTON.value)
+        
+        click_obj_by_name(ChangePasswordMenu.CHANGE_PASSWORD_CURRENT_PASSWORD_INPUT.value)
+        type(ChangePasswordMenu.CHANGE_PASSWORD_CURRENT_PASSWORD_INPUT.value, oldPassword)
+        
+        click_obj_by_name(ChangePasswordMenu.CHANGE_PASSWORD_NEW_PASSWORD_INPUT.value)
```

Comment:
Do we really need to click the input first? `type` should do the focus correctly.

Reply:
yep, not needed removed them!